core/src/main/java/com/opentext/ia/sdk/support/io/DomainObjectTooBigException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2016-2017 by OpenText Corporation. All Rights Reserved.
+ */
+package com.opentext.ia.sdk.support.io;
+
+/**
+ * Extends {@linkplain RuntimeException}. Used when a DomainObject is too big to fit in a SIP with the defined limit.
```

Comment:
No need to mention what class it extends, we can see that. Focus on the purpose, not the technicalities.